src/game/Strategic/Facts.h

```diff
@@ -1,9 +1,10 @@
 #ifndef FACTS_H
 #define FACTS_H
+#include <cstdint>
 
-enum Fact
+enum Fact : uint16_t
 {
-	FACT_NONE = -1,
+	FACT_NONE = UINT16_MAX,
```

Comment:
Did you check whether this is persisted anywhere, e.g. in save games or resource files? If so this would be a break reading from old files.

Reply:
`FACT_NONE` didn't exist in the original code base, it was added by Tron in commit 58839b9b7c633d25d500783cb8a65a32b6858af4 and is currently only used in two places to signify an invalid value. It now has the same value as NO_FACT (65535), which it replaces everywhere so that we are only using values from the Fact enum. It would also be possible to do it the other way around, keep NO_FACT and drop FACT_NONE, but this approach seems more consistent.